rabbitmq/plan.sh

```diff
@@ -1,22 +1,23 @@
 pkg_name=rabbitmq
 pkg_distname=${pkg_name}-server
 pkg_origin=core
-pkg_version=3.6.12
+pkg_version=3.7.1
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('MPL')
 pkg_description="Open source multi-protocol messaging broker"
 pkg_upstream_url="https://www.rabbitmq.com"
-pkg_source=http://www.rabbitmq.com/releases/rabbitmq-server/v${pkg_version}/rabbitmq-server-${pkg_version}.tar.xz
-pkg_shasum=c8e5a8ed5aae6167aa0050f41bd9f26d50eb16efd8abd1a1b0b01fcbccb458b9
+pkg_source=https://github.com/rabbitmq/rabbitmq-server/releases/download/v${pkg_version}/rabbitmq-server-${pkg_version}.tar.xz
+pkg_shasum=b88de3aea1ed5c4fc048cb310609fdfd430b2e4e6629e0efd9fc2a4327324fab
 pkg_dirname=${pkg_distname}-${pkg_version}
 pkg_deps=(
   core/coreutils
   core/glibc
-  core/erlang19
+  core/erlang
 )
 pkg_build_deps=(
   core/bash
   core/diffutils
+  core/elixir
```

Comment:
Purely curious, why do we need elixir here?

Reply:
@stevendanna  Looks like `rabbitmqctl` has been rewritten in Elixir in 3.7:  https://github.com/rabbitmq/rabbitmq-cli

I was hoping that would help with the [zombie-process army problem](https://github.com/habitat-sh/core-plans/issues/1099) we're seeing, but it appears to have the exact same issue.   We may have to disable or significantly alter that health_check.